cmd/monitor/import.go

```diff
@@ -27,7 +27,8 @@ import (
 	datadog "gopkg.in/zorkian/go-datadog-api.v2"
 )
 
-var inputPath string
+//var inputPath string
+var target string
```

Comment:
It looks like `target` variable is not used in this sub-command. So the variable and [flag definition in `init()`](
https://github.com/tani-yu/dogleash/blob/8dda46ff76e943bfca78f9d5955d750b472f1a45/cmd/monitor/import.go#L77-L78) should be removed.